src/LibChorus/LibChorusActivitySource.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) 2025-2025 SIL International
+// This software is licensed under the MIT License (http://opensource.org/licenses/MIT)
+
+using System.Diagnostics;
+using Chorus.VcsDrivers.Mercurial;
+
+namespace Chorus
+{
+	public static class LibChorusActivitySource
+	{
+		public const string ActivitySourceName = "SIL.LibChorus";
+		internal static readonly ActivitySource Value = new ActivitySource(ActivitySourceName);
+
+		public static void TagResumableParameters(this Activity activity, string direction, HgResumeApiParameters request)
```

Comment:
This method's callers are null checked, but I wonder if it'd be safer to (also) null check `activity` here, too.